Source/Urho3D/Core/Timer.cpp

```diff
@@ -77,7 +75,7 @@ Time::~Time()
 static unsigned Tick()
 {
 #ifdef _WIN32
-    return (unsigned)GetTickCount();
+    return (unsigned)SDL_GetTicks();
```

Comment:
`SDL_GetTicks()` is slower than `GetTickCount()`. I do not really see why switch it.

Reply:
Yeah, and `GetTickCount` has the precision of 10-16 milliseconds which is garbage.
When I want to get time in milliseconds, I sometimes want millisecond precision.
We _may_ have two separate sets of functions, but I don't think it's worth it.

Reply:
We do have two separate functions. Another one is just below - `HiresTick()`.

Reply:
So, what should I do in my code?
I have `Time`, I need to get current milliseconds with 1ms precision without sudden wraparounds.

I don't want to think now, but if you give me working code, I'll use it.